doc/sphinx/source/libCEEDapi.rst

```diff
@@ -474,6 +481,21 @@ the address of a user defined variable. Such a variable can be used later to
 explicitly wait for the completion of the operation.
 
 
+Gallery of QFunctions
+----------------------------------------
+
+LibCEED provides a gallery of built-in QFunctions in the :file:`gallery/` directory.
+The available QFunctions are the ones associated with the mass, the Laplace, and
+the identity matrices. To illustrate how the user can declare a :ref:`CeedQFunction`
```

Comment:
Matrices? or Operators?

Reply:
Ahah, I had literally used "operators" but then I thought I should be consistent with what is written earlier on in this same file (line 302 and the doc string of `tests/t500-operator.c` )

Reply:
I think when we are talking about what libCEED provides, we should say Operator. There are a handful of cases where we say 'mass matrix' or 'stiffness matrix' that feel like a reasonable exception though.

I just noticed, in the lines above, Laplace or Laplacian?

Reply:
Changed